.github/workflows/CI.yml

```diff
@@ -1,14 +1,15 @@
 name: 'CI'
+
 on: # Build any PRs and main branch changes
   workflow_dispatch: # Allows to run the workflow manually from the Actions tab
-  pull_request:
-    types:
-      - opened
-      - synchronize
-  push:
-    branches: [ master ]
-  schedule:
-    - cron: '0 0 1 * *' # Every month
+#  pull_request:
+#    types:
+#      - opened
+#      - synchronize
+#  push:
+#    branches: [ master ]
+#  schedule:
+#    - cron: '0 0 1 * *' # Every month
```

Comment:
⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️
⚠️⚠️ TO REVERT BEFORE MERGE  ⚠️⚠️
⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️